samples/feature_group/README.md

```diff
@@ -0,0 +1,108 @@
+# Feature Group Sample
+
+This sample demonstrates how to create a feature group using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.
```

Comment:
If this is taken from a notebook, can you please add a reference to it?

It'll be useful for us as well. We need to start doing it for all samples